db/migrations/20221227080309_evaluations.sql

```diff
@@ -0,0 +1,18 @@
+-- migrate:up
+CREATE TABLE evaluations (
+  id int NOT NULL AUTO_INCREMENT,
+  user_id int NOT NULL,
+  store_id INT NOT NULL,
+  rating INT NOT NULL,
+  taste INT NOT NULL,
+  price INT NOT NULL,
```

Comment:
```suggestion
  price_rate INT NOT NULL,
```
column명은 꼭 위와같을 필요는 없지만 수정이 되어야 할 것 같습니다. 우리가 의도한 기능은 가격에 대한 평점을 기록하기 위한 column이지만 이름 자체는 price다 보니 가격을 의미하는 것 같이 느껴지네요!

수정되는 이름에 따라 service와 taste도 바꿔주세요 :)